src/qtism/data/storage/xml/marshalling/Marshaller.php

```diff
@@ -471,6 +504,27 @@ public static function setXmlBase(DOMElement $element, $xmlBase)
         $element->setAttributeNS('http://www.w3.org/XML/1998/namespace', 'base', $xmlBase);
     }
 
+    /**
+     * @param BodyElement $bodyElement
+     * @param DOMElement $element
+     */
+    protected function fillBodyElementFlowTo(BodyElement $bodyElement, DOMElement $element)
+    {
+        $scan = ['aria-flowto'];
+
+        if (in_array($bodyElement->getQtiClassName(), self::$flowsToClasses, true)) {
+            array_unshift($scan, 'aria-flowsto');
```

Comment:
you expected `flowSto` here, or flowto, without S?

Reply:
okay, probably because of this: 
> Indeed, the following elements do not
                   * consider aria-flowto (the official one) but aria-flowsto which is an error: associateInteraction

Reply:
but if it is, could you add commentary here with description

Reply:
Everything is described at line 567 when calling the function.